cases/toml.test.js

```diff
@@ -79,6 +80,21 @@ describe("TOML File", () => {
       });
     });
 
+    if (testCurrency) {
+      it("selected currency is available", () => {
```

Comment:
Will this fail if there's no selected currency?  It should still fall back since i think most transfer servers will only support one currency

Reply:
This test doesn't run (nor fail) when there isn't a selected currency.

Reply:
<img width="1920" alt="Screen Shot 2020-04-20 at 6 29 48 PM" src="https://user-images.githubusercontent.com/630755/79801719-f9296880-8334-11ea-804a-8c6b7ae74309.png">